weave/trace/api.py

```diff
@@ -28,12 +28,17 @@
 from weave.trace.table import Table
 from weave.trace_server.interface.builtin_object_classes import leaderboard
 
+_global_postprocess_inputs: Callable[[Any], Any] | None = None
+_global_postprocess_output: Callable[[Any], Any] | None = None
+
 
 def init(
     project_name: str,
     *,
     settings: UserSettings | dict[str, Any] | None = None,
     autopatch_settings: AutopatchSettings | None = None,
+    global_postprocess_inputs: Callable[[Any], Any] | None = None,
```

Comment:
I feel like the `global_` prefix may be unnecessary. 

Can the `Any`s be tightened?

At a global level, I wonder if it would be useful to also pass in an Op reference to this function, so you could do something like "delete the bulky images in the inputs, but only if the Op is (some autopatched thing)"

Reply:
I added `global_` here to suggest to the user that this will be applied to EVERY op.  Do you think the doc is sufficient?

Tightened up the Anys

There's also an autopatched op config on init (that's autopatch_settings).

Reply:
I feel like the `global_` is kind of implied from this being on the weave.init. I'd probably get a second opinion but I won't block on it.

I feel like the Op is useful information, but I'm OK waiting until a customer asks for it. If we want to keep the API simple another future possibility might be a "get_current_op" kind of method.